### PR TITLE
Enhancement on the context level

### PR DESCRIPTION
**Pull request by nicolas-carlier-ls**
Few enhancement on the context level:

- Add page-token option for context list variables
- Retrieve the update-at field from Context Environment Variable

### CODE REVIEW

**Review by Kyslik — COMMENTED** [review]
Thank you, I was just going to make PR for this. 🙇🏻

File: context.go [hunk 1]
```diff
@@ -152,19 +152,24 @@ type ContextVariableList struct {
 	NextPageToken string `json:"next_page_token"`
 }
 
+type ContextListVariablesOptions struct {
+	NextPageToken string `url:"page-token,omitempty"`
```
Comment (Kyslik) [inline context.go:156]:
```suggestion
	PageToken string `url:"page-token,omitempty"`
```

Consistency across other `*Options` like `ProjectListPipelinesOptions` etc.

Comment (nicolas-carlier-ls) [reply context.go]: Indeed, fixed.

**Review by grezar — APPROVED** [review]
Thanks!
I'll release this with #31

File: context.go [hunk 2]
```diff
@@ -152,19 +152,24 @@ type ContextVariableList struct {
 	NextPageToken string `json:"next_page_token"`
 }
 
+type ContextListVariablesOptions struct {
+	PageToken string `url:"page-token,omitempty"`
```
Comment (Kyslik) [inline context.go:156]: Should have been `*string` instead. Fixing in #32.

**Pull request merged**